lightning/src/ln/channelmanager.rs

```diff
@@ -572,13 +572,16 @@ pub(crate) const MAX_LOCAL_BREAKDOWN_TIMEOUT: u16 = 2 * 6 * 24 * 7;
 // i.e. the node we forwarded the payment on to should always have enough room to reliably time out
 // the HTLC via a full update_fail_htlc/commitment_signed dance before we hit the
 // CLTV_CLAIM_BUFFER point (we static assert that it's at least 3 blocks more).
-pub const MIN_CLTV_EXPIRY_DELTA: u16 = 6 * 6;
+pub const MIN_CLTV_EXPIRY_DELTA: u16 = 6*7;
```

Comment:
Should we modify the default config value which is of 72 ? What's the rational to have differing default config value and implementation lower bound, cautiousness ?

Reply:
The rationale is that I forgot to update the config :).

Reply:
Oh, I take that back, the config looks good? Config default is 12 hours, not the 6/7 here, I don't think we need to change that.

Reply:
I'm good with the cautiousness argument to justify config default of 12 hours :)